### PR TITLE
Html email templates

### PR DESCRIPTION
**Pull request by atoncetti**
Implementation of [#68](https://github.com/matchd-ch/stories/issues/68)

### CODE REVIEW

**Comment by atoncetti** [comment]
How to debug/test the templates:

Create a file `matchd-backend/db/view/test_view.py`
File contents:
 ```
from django.shortcuts import render

def test_view(request):
    return render(request, 'db/email/activation/body.html', context={'user': {'type': 'student', 'first_name': 'TEST'}})
```


 Inside `urls.py` add:
 ```
from db.view.test_view import test_view

...

url_patterns = [
   ...
   path('test/', test_view)
]

```


For changing the linked css stylesheed:
```
{% extends '../base/base.html' %}

{% load static %}
{% block css %}
    <link rel="stylesheet" href="{% static 'css/email/my_new_style.css' %}">
    {{ block.super }}  <!-- This line includes the stylesheet from the 'parent' template too -->
{% endblock %}
```

Now you can reach the route at `api.matchd.localhost:8000/test/' to see the email

File: db/templates/db/email/base/base.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {% block css %}
+        <link rel="stylesheet" href="{{ 'static/css/email/style.css' }}">
```
Comment (maennchen) [inline db/templates/db/email/base/base.html:5]: ?

**Comment by atoncetti** [comment]
TODO: Revert pr branch to main

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ jobs:
   test:
     name: "Test"
 
-    uses: matchd-ch/matchd-backend/.github/workflows/part_test.yml@main
+    uses: matchd-ch/matchd-backend/.github/workflows/part_test.yml@feat/html_emails
```
Comment (maennchen) [inline .github/workflows/pr.yml:12]: TODO before merge: revert back to `main`

**Pull request merged**